dist/common/scripts/scylla-housekeeping

```diff
@@ -172,6 +179,7 @@ parser.add_argument('-c', '--config', default="", help='An optional config file.
 parser.add_argument('--uuid', default="", help='A uuid for the requests')
 parser.add_argument('--uuid-file', default="", help='A uuid file for the requests')
 parser.add_argument('--repo-files', default="", help='The repository files that is been used for private repositories')
+parser.add_argument('--env-file', default="/etc/scylla.d/housekeeping.env", help='an env file, allows to check version for a specific environment like AWS or GCP')
```

Comment:
"Environment" could also mean shell environment variables, for example. Perhaps call this "--platform-file" or something like that? @bentsi, what do you think?

Reply:
I'm not crazy about env also, we tried to think of a better name, I'll change it to platform-file